centaur/src/main/resources/standardTestCases/hello_private_repo.test

```diff
@@ -0,0 +1,15 @@
+name: hello_private_repo
```

Comment:
It's not clear to me, so just to as an obvious question: Does this new test fail without the change to `StandardLifecycleActorFactory`?

Reply:
Yes the test will fail without these changes. The workflow would succeed without these changes, but without the added plumbing through of Docker credentials the Cromwell engine will be unable to obtain the hash for the Docker image. Consequently the call caching expected by the`testFormat: runtwiceexpectingcallcaching` will not happen and the test will fail.